packages/slice-machine/lib/models/common/widgets/UID/index.ts

```diff
@@ -23,24 +23,20 @@ import {
 } from "@prismicio/types-internal/lib/customtypes/widgets";
 
 const FormFields = {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
   label: DefaultFields.label,
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
   id: {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
     ...DefaultFields.id,
     validate: {
```

Comment:
if it's disabled, do we still need the validate function ?

Reply:
no not needed since it's not really used. But if I remove it it would still be present (it would just be the default one coming from `... DefaultFields.id`). So I though it'd be best to have an unused-correct validation than an unused-incorrect validation. 

Mostly to avoid potential bugs, if the disabled field is broken/removed by accident, we want to still have the safety.

Reply:
right 👍